fusor-ember-cli/app/controllers/configure-environment.js

```
@@ -65,6 +65,7 @@ export default Ember.Controller.extend(ConfigureEnvironmentMixin, NeedsDeploymen
         self.get('lifecycleEnvironments').addObject(result._internalModel);
         self.set('selectedEnvironment', environment);
         self.get('deploymentController.model').set('lifecycle_environment', environment);
+        self.get('deploymentController').set('errorMsg', null);
```

this hides the error message on success after getting a previous error message.